package.yaml

```diff
@@ -8,25 +8,19 @@ copyright:           "Sibi Prabakaran"
 
 extra-source-files:
 - README.md
-- ChangeLog.md
-
-data-files:
-- faker/lib/locales/*yml
-- faker/lib/locales/en/*yml
-- customFakeSource/en/*yml
 
 # Metadata used when publishing your package
-synopsis:            Library for producing fake data
+synopsis:            Library for streamly and bytestring interoperation. 
 category:            Streamly, Stream
 
 # To avoid duplicated efforts in documentation and dealing with the
 # complications of embedding Haddock markup inside cabal files, it is
 # common to point users to the README.md file.
-description:         Please see the README on GitHub at <https://github.com/psibi/fakedata#readme>
+description:         Please see the README on GitHub at <https://github.com/psibi/streamly-bytestring#readme>
 
 dependencies:
-- base >= 4.7 && < 5
-- streamly
+- base       >= 4.7 && < 5
+- streamly   >= 0.6.0
 - bytestring
 - exceptions
```

Comment:
I think we can remove exceptions dependency.

Reply:
Also,
`streamly >= 0.7.0 && < 0.8`
0.6.0, 0.6.1 doesn't have the required modules